app/forms/waste_exemptions_engine/exemptions_form.rb

```diff
@@ -7,10 +7,9 @@ class ExemptionsForm < BaseForm
     validates :exemptions, "waste_exemptions_engine/exemptions": true
 
     def submit(params)
-      # Rails authomatically delete params for which the value is empty :/
-      params ||= {}
+      params[:exemption_ids] ||= []
```

Comment:
This line here is in order to by-pass the Rails 4 bug for which empty parameters are parsed as `nil`. This is fixed in Rails 5, so whenever we upgrade we can also get rid of this

Reply:
I think it would help then to leave some comment to that affect. When this gets squashed the only place that will note that is this PR.

Reply:
```suggestion
      # TODO: This line here is in order to by-pass the Rails 4 bug for which empty parameters are parsed as nil. 
      # This is fixed in Rails 5, so whenever we upgrade we can also get rid of this
      params[:exemption_ids] ||= []
```